src/classes/ReciprocalRelHealthCheckVMapper.cls

```diff
@@ -0,0 +1,533 @@
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @date 2021
+* @group Health Check
+* @group-content ../../ApexDocContent/HealthCheck.htm
+* @description View Model Mapper providing a Health Check Group View Model
+* for Reciprocal Relationship Mappings and Settings.
+*/
+public virtual with sharing class ReciprocalRelHealthCheckVMapper {
+
+    /***************************************************************************
+    * @description Instance for Singleton Pattern
+    ******************************************************************************/
+    @TestVisible
+    private static ReciprocalRelHealthCheckVMapper instance;
+
+    /***************************************************************************
+    * @description Empty constructor is not public for Singleton Pattern
+    ***************************************************************************/
+    protected ReciprocalRelHealthCheckVMapper() {}
+
+    /***************************************************************************
+    * @description Static method to get the current instance for Singleton
+    * pattern
+    * @return The instance of ReciprocalRelHealthCheckVMapper.
+    ***************************************************************************/
+    public static ReciprocalRelHealthCheckVMapper getInstance() {
+
+        if (instance == null) {
+            instance = new ReciprocalRelHealthCheckVMapper();
+        }
+
+        return instance;
+    }
+
+    /***************************************************************************
+    * @description Provides comprehensive Health Check Group View Model for
+    * Reciprocal Relationship Settings
+    * @return The Health Check Group View Model for Reciprocal Relationship
+    * Settings
+    ***************************************************************************/
+    public virtual HealthCheckGroupVModel getHealthCheckGroup() {
+        HealthCheckGroupVModel healthCheckGroupVModel = new HealthCheckGroupVModel(
+            Label.stgHealthCheckReciprocalRelTitle,
+            Label.stgHealthCheckStatusPassed,
+            0,
+            0,
+            new List<String>(),
+            new List<HealthCheckItemVModel>()
+        );
+
+        List<RelationshipLookupModel> relationshipLookupModelList =
+            this.locateRelationshipLookupSettingsService(
```

Comment:
I know we are trying to follow a new code style and it is probably fine if that works for the team. However, do we really want to put the parentheses into a separate line if there is no parameter? Also, we really need to update our code style otherwise inconsistency may impede our code review.

Reply:
This was largely a means to try to break up those lines to not be massive. We'll address this soon and in a consistent format with prettier, assuming the team wants it.